packages/components/organisms/f-form/package.json

```diff
@@ -40,7 +40,9 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.7.0"
+    "@justeat/f-services": "1.7.0",
+    "@justeat/f-button": "3.0.2",
+    "@justeat/f-form-field": "4.0.1"
```

Comment:
We've started to make any subcomponents like `f-button` and `f-form-field` peerDependencies which then get pulled in at the page component level (so in `f-checkout` then for instance). Essentially f-button and f-form-field would be included as a `peerDependency` (so they don't get built as part of the f-form component) and also included as a `devDependency` (so that the tests can still run with no problems).

You can see how to do this by looking at a PR [like this one](https://github.com/justeat/fozzie-components/pull/1546/files) (as you'll need to add a tiny bit of config as well so that the peerDependency is kept external when the component is built). Give me a msg if you have any questions around this though and can pair on it.